README.md

```diff
@@ -1 +1,97 @@
-# weatherapp
\ No newline at end of file
+# Weather App
+
+ This weather App is created to show a list of weather informations for different cities in Nigeria.
+
+## Built With
+
+* HTML
+* CSS
+* JS
+
+## Framework
+
+* React
+
+## Getting Started
+
+Tools Needed
+
+* Text editor
+* Node.js
+* Gitflow
+
+Clone this repository on to your local PC
+
+* Click on the Code green button
+* By the right end of the read-only input containing the repository link click the clipboard icon to copy the link
+* In your local PC, open your terminal in the folder you would like to clone the repository into
+* Clone the repository with the command: git clone (copied link); like so: <https://github.com/OyePriscilla/weatherapp>
+* After the clone, type in the command cd weatherapp to access the directory on the terminal.
+
+* Once the files are on your machine, open the weatherapp folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up
+
+## 🛠 Installation & Set Up
+
+### Create React App
+
+  npx create-react-app my-react-app
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
```

Comment:
- [ ] Please remove these things because you are not gonna need it anymore :+1:

Reply:
Thanks @HAMMAS-SALEEM  changes effected!!!